FindOnPage.md

```diff
@@ -0,0 +1,382 @@
+```cpp
+//! [StopFind]
+void AppWindow::StopFind()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+    
+    // Stop the find operation
+    find.StopFind();
+}
+//! [StopFind]
+```
+
+### Retrieve the Number of Matches
+
+#### Description
+To retrieve the total number of matches found during a find operation within a WebView2 control, developers can utilize the `GetMatchCount` method.
+
+
+```cpp
+//! [GetMatchCount]
+void AppWindow::GetMatchCount()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Get the match count
+    LONG matchCount;
+    find.GetMatchesCount(&matchCount);
+
+    // Update UI or handle matchCount
+}
+//! [GetMatchCount]
+```
+
+### Retrieve the Index of the Active Match
+
+#### Description
+Developers can retrieve the index of the currently active match within a WebView2 control using the `GetActiveMatchIndex` method.
+
+
+```cpp
+//! [GetActiveMatchIndex]
+void AppWindow::GetActiveMatchIndex()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Get the active match index
+    LONG activeMatchIndex;
+    find.GetActiveMatchIndex(&activeMatchIndex);
+
+    // Update UI or handle activeMatchIndex
+}
+//! [GetActiveMatchIndex]
+```
+
+### Navigate to the Next Match
+
+#### Description
+To navigate to the next match found during a find operation within a WebView2 control, developers can use the `FindNext` method.
+
+
+```cpp
+//! [FindNext]
+void AppWindow::FindNext()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Find the next occurrence
+    find.FindNext();
+}
+//! [FindNext]
+```
+
+### Navigate to the Previous Match
+
+#### Description
+To navigate to the previous match found during a find operation within a WebView2 control, developers can use the `FindPrevious` method.
+
+
+```cpp
+//! [FindPrevious]
+void AppWindow::FindPrevious()
+{
+    // Get the find interface
+    auto find = webView2Control.GetFind();
+
+    // Find the previous occurrence
+    find.FindPrevious();
+}
+//! [FindPrevious]
+```
+
+#### Handle Match Count Changes
+
+```cpp
+void OnMatchCountChanged(LONG matchesCount)
+{
+    // Handle match count changes
+    // Update UI elements or perform actions based on the new match count
+}
+```
+#### Handle Active Match Index Changes
+```cpp
+void OnActiveMatchIndexChanged(ICoreWebView2* sender, ICoreWebView2StagingFindActiveMatchIndexChangedEventArgs* args)
+{
+    // Handle active match index changes
+    // Update UI to reflect the change in the active match index
+}
+```
+
+#### Handle Find Operation Completion
+
+```cpp
+void OnFindOperationCompleted(HRESULT value, LONG activeMatchIndex, LONG matchesCount)
+{
+    // Handle find operation completion
+    // Update UI elements, display search results, or handle errors
+}
+```
+## API Details
```

Comment:
Can you include something about timing in the docs? Can there only be one find operation in progress at a time per WebView?